style.css

```diff
@@ -0,0 +1,40 @@
+body,html{
+    padding: 0px;
+    margin: 0px;
+    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+.forms{
+    box-shadow: 2px 2px 2px 2px rgba(0,0,0,0.3);
```

Comment:
Só para informar: quando o valor for inferior a 1, podemos escrever o decimal sem o zero à esquerda do ponto. Assim: `.3`